src/fixtures/runCustomFixtures.ts

```diff
@@ -24,24 +24,25 @@ export interface CustomFixture {
 
 export const runCustomFixtures = ({ targetId, clientId, tests }: CustomFixture) => {
   describe(`custom fixtures for ${targetId}:${clientId}`, () => {
-    tests.forEach(({ it: title, expected: fixtureFile, options, input: request }) => {
```

Comment:
Had to restructure all of these tests into formal `describe.each` and `it.each` blocks because with the async change Vitest wasn't picking them up as they were getting invoked after Vitest was expecting them to because it was running an async test inside of a non-async foreach loop.